wiki/Client/Menu_tips/de.md

```diff
@@ -92,24 +90,36 @@ Diese Tipps erscheinen im [Beatmap-Editor](/wiki/Client/Beatmap_editor) unter de
 
 ## osu!(lazer)
 
-Diese Tipps erscheinen beim Start von [osu!(lazer)](/wiki/Client/Release_stream/Lazer), vor der Intro-Sequenz. Für diesen Artikel wurden die Tipps übersetzt, da sie im Spiel nur auf Englisch verfügbar sind.
+Diese Tipps erscheinen unten im Hauptmenü von [osu!(lazer)](/wiki/Client/Release_stream/Lazer). Für diesen Artikel wurden die Tipps übersetzt, da sie im Spiel nur auf Englisch verfügbar sind.
 
-- Du kannst überall `Strg` + `T` drücken, um die Toolbar zu aktivieren oder zu deaktivieren!
+- Du kannst überall `Strg` + `T` drücken, um die Symbolleiste zu aktivieren oder zu deaktivieren!
 - Du kannst überall `Strg` + `O` drücken, um auf die Optionen zuzugreifen!
-- Alle Einstellungen sind dynamisch und werden in Echtzeit wirksam. Probiere, das Spiel zu pausieren und den Skin zu ändern!
+- Alle Einstellungen sind dynamisch und werden in Echtzeit wirksam. Probiere, den Skin während des Spiels zu ändern!
```

Comment:
The EN version changes from "while playing" to "while watching autoplay". I think we should reflect that here

Reply:
Even though it is obscured and not directly mentioned, this line refers to the `Autoplay` mod. Taking some freedom in translation here is not problematic in my opinion.

My other suggestion would be

```suggestion
- Alle Einstellungen sind dynamisch und werden in Echtzeit wirksam. Probiere, den Skin im automatischen Spielmodus zu ändern!
```

or even more directly

```suggestion
- Alle Einstellungen sind dynamisch und werden in Echtzeit wirksam. Probiere, den Skin zu ändern während die Mod `Autoplay` aktiv ist!
```

Reply:
> Even though it is obscured and not directly mentioned, this line refers to the Autoplay mod. Taking some freedom in translation here is not problematic in my opinion.

I'm not against this in general, but in this specific case the English version was changed to explicitly mention the autoplay mod. So I think also mentioning it in the translation is preferable.

Of your two suggestions, the second one feels better IMHO